src/SFA.DAS.PR.Application/Requests/Commands/CreatePermissionRequest/CreatePermissionRequestCommandValidator.cs

```diff
@@ -0,0 +1,32 @@
+﻿using FluentValidation;
+using SFA.DAS.PR.Application.Common.Validators;
+using SFA.DAS.PR.Domain.Interfaces;
+
+namespace SFA.DAS.PR.Application.Requests.Commands.CreatePermissionRequest;
+
+public class CreatePermissionRequestCommandValidator : AbstractValidator<CreatePermissionRequestCommand>
+{
+    public static readonly string UkprnValidationMessage = "A Ukprn must be provided.";
+
+    public CreatePermissionRequestCommandValidator(
+        IRequestReadRepository requestReadRepository,
+        IProviderReadRepository providerReadRepository,
+        IAccountLegalEntityReadRepository accountLegalEntityReadRepository
+    )
+    {
+        RuleFor(x => x.Ukprn)
+            .NotEmpty()
+            .WithMessage(UkprnValidationMessage);
+
+        RuleFor(x => x.Ukprn)
+            .IsValidUkprn(providerReadRepository);
+
+        RuleFor(a => a.AccountLegalEntityId)
+            .ValidateAccountLegalEntityExists(accountLegalEntityReadRepository);
+
+        RuleFor(a => new RequestValidationObject() { Ukprn = a.Ukprn, AccountLegalEntityId = a.AccountLegalEntityId })
+            .ValidateRequest(requestReadRepository);
+
+        RuleFor(a => a.Operations).ValidateOperationCombinations();
```

Comment:
The combination message for the operations, needs a little bit of improving as the comma delimiter is used between and within combinations.